protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3115,6 +3115,8 @@ message ListPaidLiquidityFeesRequest {
   repeated string party_ids = 4;
   // Pagination controls.
   optional Pagination pagination = 5;
+  // Whether to return all derived parties from AMMs for the given party.
```

Comment:
```suggestion
  // Whether to return all derived parties from AMMs for the given party. If used, party ID is required.
```

Gave this some thought and I think since we have the extra note in GraphQL, it could be useful here too.

Reply:
updated